proteinflow/data/__init__.py

```diff
@@ -294,7 +294,7 @@ def get_sequence(self, chains=None, encode=False, cdr=None, only_known=False):
         if cdr is not None:
             assert cdr in CDR_REVERSE, f"CDR must be one of {list(CDR_REVERSE.keys())}"
         chains = self._get_chains_list(chains)
-        seq = "".join([self.seq[c] for c in chains])
+        seq = "".join([self.seq[c] for c in chains]).replace("B", "")
```

Comment:
why is that?

Reply:
just a bug in one fasta file in brenda, where the sequence starts with B